constants_local.xml

```diff
@@ -43,6 +50,7 @@
     </smart_contract>
     <accounts>
         <account>
+			<private_key>FFFE8A87A980C0F32F0EDF2659684292B39D3CE7511EDE66CE6B148451739D5F</private_key>
```

Comment:
Should we add this private key in the main rep ?

Reply:
That's an accident, we will change that key pair however